website/static/js/profile.js

```diff
@@ -739,8 +742,19 @@ var SocialViewModel = function(urls, modes, preventUnsaved) {
 SocialViewModel.prototype = Object.create(BaseViewModel.prototype);
 $.extend(SocialViewModel.prototype, SerializeMixin.prototype, TrackedMixin.prototype);
 
+const arrayInDBKeys = ['twitter', 'linkedIn', 'github'];
```

Comment:
Nit: We're not using Babel to transpile ES2015, so we've just been using `var` (though it's not a huge deal, since `const` is [supported in IE11](https://caniuse.com/#feat=const)). We also use UPPER_SNAKE_CASE for constants.